app/controllers/lists_controller.rb

```diff
@@ -4,9 +4,15 @@ class ListsController < ApplicationController
   def create
     @list = current_user.lists.new(name: params[:name])
     if @list.save
-      redirect_to root_path
+      respond_to do |format|
+        format.html { redirect_to root_path(), notice: "List created" }
```

Comment:
Minor nit: parents aren’t needed on root_path.
```suggestion
        format.html { redirect_to root_path, notice: "List created" }
```